jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -113,7 +115,14 @@ static Path getExplodedWarDirectory(Project project) {
     this.project = project;
     this.javaLayerConfigurations = javaLayerConfigurations;
 
-    ansiLoggerWithFooter = new AnsiLoggerWithFooter(logger::lifecycle);
+    // TODO: Make SHOW_PROGRESS be true by default.
```

Comment:
Maybe we could make it `HIDE_PROGRESS` so it can be false by default?

Reply:
This system property is currently for hiding the feature behind it for testing before actually enabling it for release, so we can change to `HIDE_PROGRESS` when ready.

Reply:
It would be worth pulling the enablement into a method. You'll want to check that either we're on Windows (CMD windows support ANSI), or `TERM` matches `xterm.**` or `vt[0-9]*`, or at the very worst `TERM` is not null and `TERM != 'dumb'`.

Reply:
Sounds good - moving to separate method and adding this check.

Reply:
If the user is redirecting all stdin and stdout to a file or the underlying logging system is configured to send output to a file, the `TERM` env variable will be meaningless, but I guess we won't worry about that too much.

Reply:
Yea, not sure how we could detect if stdout is redirected to a file in Java without calling `isatty`.

Reply:
Maven also supports a _batch_ mode (`-B`)

Reply:
I think we can ignore batch mode since that only affects stdin.

Reply:
I tried running Jib with `-B` and although Maven's own colors were disabled, Jib's colors were not.